modules/apps/object/object-test/src/testFunctional/macros/ObjectModelBuilder.macro

```diff
@@ -1,5 +1,51 @@
 definition {
 
+	@summary = "Default summary"
+	macro addObjectRelationshipViaUIInModelBuilder(invalidEntry = null, parameterOption = null, name = null, label = null, type = null, object = null,) {
```

Comment:
@thaiscabrals I think we can remove viaUI here since we are already stating that it is being done in the Model Builder. Let me know what you think, thanks!

Reply:
Hey @thaiscabrals it looks like you're making fixes to the previously created macro in this commit. You should be making those changes in the original commit instead of fixing them in a later commit to keep the history clean. You can use interactive rebase to fix this.